samples/oauth_ccg_postman_pre_request_script/CCG Pre-request Script.postman_collection.json

```diff
@@ -0,0 +1,215 @@
+{
+	"info": {
+		"_postman_id": "4a896832-c77a-46b1-b107-0528de527bb7",
+		"name": "CCG Pre-request Script",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "30064595"
+	},
+	"item": [
+		{
+			"name": "load jsrsasign-js",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.globals.set(\"jsrsasign-js\", responseBody)"
```

Comment:
This is a get request that sets a global var, can this just be a function in the main script that runs if that var is not set?

sudo code:

```
function setup() {
    if (pm.globals.has("jsrsasign-js"))
        return;
    pm.sendRequest("http://kjur.github.io/jsrsasign/jsrsasign-latest-all-min.js", (err, res) => {
            console.log("getting jsrsasign-js");
            pm.globals.set("jsrsasign-js", res.text());
            eval(pm.globals.has("jsrsasign-js"));
        }
    );
}
```

Reply:
hint: https://blog.postman.com/adding-external-libraries-in-postman/